### ISSUE TITLE
Rearranging the images in the gallery

### ISSUE THREAD

**Issue by srikat** [xwp/wp-customize-image-gallery-control#7]
Is there a way to rearrange the selected gallery images via drag and drop?

**Comment by grappler** [comment]
@srikat Yes, it is possible to drag and drop the images in a different order. It is just not very intuitive due to the styling.
